pshtt/pshtt.py

```diff
@@ -594,14 +605,34 @@ def https_check(endpoint):
 
     try:
         cert_plugin_result = None
-        command = sslyze.plugins.certificate_info_plugin.CertificateInfoScanCommand(ca_file=CA_FILE)
-        scanner = sslyze.synchronous_scanner.SynchronousScanner()
-        cert_plugin_result = scanner.run_scan_command(server_info, command)
+        scanner = Scanner()
+        command = ScanCommand.CERTIFICATE_INFO
+        if CA_FILE is not None:
+            command_extra_args = {
+                ScanCommand.CERTIFICATE_INFO: CertificateInfoExtraArguments(custom_ca_file=Path(CA_FILE))
```

Comment:
`command` is somewhat superfluous now that it is not something we have to create, but if we're using it, then it makes sense to tie it in here at least.

```suggestion
                command: CertificateInfoExtraArguments(custom_ca_file=Path(CA_FILE))
```